tests/test_docs/test_orm_integration/test_orm_integration.py

```diff
@@ -0,0 +1,224 @@
+
+
+ORM_SELLER_STRATEGY_PATH = Path(
+    ROOT_DIR, "tests", "test_docs", "test_orm_integration", "orm_seller_strategy.py"
+)
+
+
+class TestOrmIntegrationDocs(AEAWithOefTestCase):
+    """This class contains the tests for the orm-integration.md guide."""
+
+    @skip_test_ci
+    def test_orm_integration_docs_example(self, pytestconfig):
+        """Run the weather skills sequence."""
+        self.initialize_aea()
+
+        seller_aea_name = "my_seller_aea"
+        buyer_aea_name = "my_buyer_aea"
+        self.create_agents(seller_aea_name, buyer_aea_name)
+
+        ledger_apis = {FETCHAI: {"network": "testnet"}}
+
+        # Setup seller
+        seller_aea_dir_path = Path(self.t, seller_aea_name)
+        os.chdir(seller_aea_dir_path)
+        self.add_item("connection", "fetchai/oef:0.2.0")
+        self.add_item("skill", "fetchai/generic_seller:0.1.0")
+        self.run_install()
+        force_set_config("agent.ledger_apis", ledger_apis)
+        self.set_config("agent.default_connection", "fetchai/oef:0.2.0")
+
+        # Setup Buyer
+        buyer_aea_dir_path = Path(self.t, buyer_aea_name)
+        os.chdir(buyer_aea_dir_path)
+
+        self.add_item("connection", "fetchai/oef:0.2.0")
+        self.add_item("skill", "fetchai/generic_buyer:0.1.0")
+        self.run_install()
+        force_set_config("agent.ledger_apis", ledger_apis)
+        self.set_config("agent.default_connection", "fetchai/oef:0.2.0")
+
+        # Generate and add private keys
+        self.generate_private_key()
+        self.add_private_key()
+
+        # Add some funds to the buyer
+        self.generate_wealth()
+
+        # Update the seller AEA skill configs.
+        seller_skill_config_path = Path(
+            seller_aea_dir_path,
+            "vendor",
+            "fetchai",
+            "skills",
+            "generic_seller",
+            DEFAULT_SKILL_CONFIG_FILE,
+        )
+        seller_skill_config = yaml.safe_load(seller_skill_config_path.open())
+        seller_skill_config.update(yaml.safe_load(seller_strategy_replacement))
+        yaml.safe_dump(seller_skill_config, seller_skill_config_path.open("w"))
+
+        # Update the buyer AEA skill configs.
+        buyer_skill_config_path = Path(
+            buyer_aea_dir_path,
+            "vendor",
+            "fetchai",
+            "skills",
+            "generic_buyer",
+            DEFAULT_SKILL_CONFIG_FILE,
+        )
+        updated_skill_config = yaml.safe_load(buyer_skill_config_path.open())
+        updated_skill_config.update(yaml.safe_load(buyer_strategy_replacement))
+        yaml.safe_dump(updated_skill_config, buyer_skill_config_path.open("w"))
+
+        # Replace the seller strategy
+        seller_stategy_path = Path(
+            seller_aea_dir_path,
+            "vendor",
+            "fetchai",
+            "skills",
+            "generic_seller",
+            "strategy.py",
+        )
+        seller_stategy_path.write_text(ORM_SELLER_STRATEGY_PATH.read_text())
```

Comment:
Should we expose a method on the test case to this effect? I.e. for a generic set of paths? `replace_file_content(source_path, target_path)`?